tonic/datasets/s_mnist.py

```diff
@@ -0,0 +1,141 @@
+import os
+import numpy as np
+from struct import unpack
+from .dataset import Dataset
+from .download_utils import (
+    download_and_extract_archive,
+    extract_archive,
+)
+
+class SMNIST(Dataset):
+    """Spiking sequential MNIST"""
+
+    base_url = "https://storage.googleapis.com/cvdf-datasets/mnist/"
+    train_images_file = "train-images-idx3-ubyte"
+    train_labels_file = "train-labels-idx1-ubyte"
+    test_images_file = "t10k-images-idx3-ubyte"
+    test_labels_file = "t10k-labels-idx1-ubyte"
+    sensor_size = (99,)
+    ordering = "txp"
+
+    classes = [
+        "0 - zero",
+        "1 - one",
+        "2 - two",
+        "3 - three",
+        "4 - four",
+        "5 - five",
+        "6 - six",
+        "7 - seven",
+        "8 - eight",
+        "9 - nine"]
+
+    def __init__(self, save_to, train=True, duplicate=True, dt=1000.0, 
+                 download=True, transform=None, target_transform=None):
+        super(SMNIST, self).__init__(save_to, transform=transform, 
+                                     target_transform=target_transform)
+        self.location_on_system = save_to
```

Comment:
could you replace this with a `os.path.join(save_to, "smnist")` please? So that all files are stored in this subfolder